src/obfl-specification.html

```diff
@@ -1238,7 +1240,7 @@ <h4 id="L1353" class="include">marker</h4>
 <div class="sidebar markup">
 <dl>
   <dt>Usage</dt>
-    <dd>before, after, toc-entry, block, item</dd>
+    <dd>before, after, toc-entry, block, toc-block, item</dd>
```

Comment:
If I understood this definition correctly this is not correct.

```marker``` may not be used directly inside of a toc-block